SPECS-EXTENDED/capstone/capstone.spec

```diff
@@ -1,230 +1,327 @@
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Name:           capstone
-Version:        4.0.2
-Release:        4%{?dist}
+Version:        5.0.1
+Release:        1%{?dist}
 Summary:        A lightweight multi-platform, multi-architecture disassembly framework
 
-License:        BSD
+License:        BSD-3-Clause AND BSD-4-Clause AND APSL-2.0 AND NCSA AND MIT
 URL:            http://www.capstone-engine.org/
 #               https://github.com/aquynh/capstone/releases
-Source0:        https://github.com/aquynh/capstone/archive/%{version}.tar.gz#/%{name}-%{version}.tar.gz
-
-# Test suite binary samples to be used for disassembly
-# Source1:
-
-# Fedora 29 makes python executable separate from python2 and python3. This patch makes
-# it possible to specify PYTHON2 and PYTHON3 binary to be explicit that by "python" we mean "python2"
-# Patch0:         capstone-python.patch
-
-# Upstream patch which fixes libcapstone.pc.
-# See: https://github.com/aquynh/capstone/issues/1339
-# Patch1:         0001-Fix-include-path-in-pkg-config-for-Makefile-too-1339.patch
-
+VCS:            https://github.com/capstone-engine/capstone/
+ 
 %global         common_desc %{expand:
 Capstone is a disassembly framework with the target of becoming the ultimate
 disasm engine for binary analysis and reversing in the security community.}
 
-# Build with python3 package by default
-%bcond_without  python3
-
-# Build without python2 package for newer releases f32+ and rhel8+
-
-%bcond_with     python2
-
-
-
 
 
+ 
+%define _lto_cflags %{nil}
+#%%global _hardened_build 1
+ 
+%global         gituser         capstone-engine
+%global         gitname         capstone
+# 5.0.1 release
+%global         gitdate         20230823
+%global         commit          097c04d9413c59a58b00d4d1c8d5dc0ac158ffaa
+%global         shortcommit     %(c=%{commit}; echo ${c:0:7})
+ 
+# Source0:      https://github.com/%%{gituser}/%%{gitname}/archive/%%{commit}/%%{name}-%%{version}-%%{shortcommit}.tar.gz
+Source0:        https://github.com/%{gituser}/%{gitname}/archive/%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

Comment:
Look into the PR check failure for Cgmanifest. The source URL does not match the one in cgmanifest.json